src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/fixtures/WiremockFixtures.java

```diff
@@ -134,6 +133,13 @@ public static void stubPatchDocumentMetaData(StoredDocumentHalResource response)
                                     .withHeader(CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)));
     }
 
+    public static void stubGetGreeting() {
+        stubFor(WireMock.get(urlPathEqualTo("/greeting"))
```

Comment:
do we need this? i mean test itself

Reply:
This is part of another change I'm yet to push - please ignore it for the moment